content/it/docs/concepts/overview/kubernetes-api.md

```diff
@@ -0,0 +1,126 @@
+---
+title: Le API di Kubernetes 
+content_template: templates/concept
+weight: 30
+card: 
+  name: concepts
+  weight: 20
+---
+
+{{% capture overview %}}
+
+Le convenzioni generali seguite dalle API sono descritte in [API conventions doc](https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md).
+
+Gli *endpoints* delle API, la lista delle risorse esposte ed i relativi esempi sono descritti in [API Reference (EN)](/docs/reference).
+
+L'accesso alle API da remoto è discusso in [Controlling API Access doc (EN)](/docs/reference/access-authn-authz/controlling-access/).
+
+Le API di Kubernetes servono anche come riferimento per lo schema dichiarativo della configurazione del sistema stesso. Il comando [kubectl](/docs/reference/kubectl/overview/) può essere usato per creare, aggiornare, cancellare ed ottenere le istanze delle risorse esposte attraverso le API.
+
+Kubernetes persiste il suo stato (al momento in [etcd](https://coreos.com/docs/distributed-configuration/getting-started-with-etcd/)) usando la rappresentazione delle risorse implementata dalle API.
```

Comment:
"persiste il suo stato" feels weird. What do you think of "assicura la persistenza del suo stato"?